integration-tests/terraform/common/variables.tf

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+variable "vpc_security_group" {
```

Comment:
Why is this a variable? I dont see it being set.

Reply:
changed it in revision